src/components/Reports/FinancialAccountsReport/AccountSummary/AccountSummary.tsx

```diff
@@ -0,0 +1,384 @@
+      appendCategoryToCategoriesArray({
+        categories: item.creditByCategories,
+        categoryArray: creditsCategories,
+        index: idx,
+      });
+
+      // Debits Categories
+      appendCategoryToCategoriesArray({
+        categories: item.debitByCategories,
+        categoryArray: debitsCategories,
+        index: idx,
+      });
+    });
+
+    return {
+      closingBalances,
+      credits,
+      creditsCategories: creditsCategories.sort((a, b) =>
+        a.name.localeCompare(b.name),
+      ),
+      debits,
+      debitsCategories: debitsCategories.sort((a, b) =>
+        a.name.localeCompare(b.name),
+      ),
+      openingBalances,
+      periods,
+      surplus,
+    };
+  }, [data]);
+
+  return (
+    <>
+      <FinancialAccountHeader />
+      <Divider />
+      {!data && (
+        <Box
+          display="flex"
+          justifyContent="center"
+          alignItems="center"
+          height="100%"
+        >
+          <CircularProgress data-testid="LoadingRC" />
+        </Box>
+      )}
+
+      {data && (
+        <Box pl={2} pr={2}>
+          <TableContainer sx={{ marginBottom: 6 }}>
+            <Table sx={{ minWidth: 650 }} aria-label="table">
+              <TableHead>
+                <TableRow>
+                  <TableCell>
+                    <Typography variant="h6">{t('Category')}</Typography>
+                  </TableCell>
+                  {tableData.periods?.map((period, idx) => {
+                    return (
+                      <TableCell
+                        key={`startDate-${idx}-${period?.startDate}`}
+                        align="right"
+                      >
+                        {period.startDateFormatted}
+                      </TableCell>
+                    );
+                  })}
+                </TableRow>
+              </TableHead>
+
+              <TableBody>
+                {/* Opening Balance */}
+                <TableRow>
+                  <StyledTableCell>
+                    <Typography variant="body1">
+                      {t('Opening Balance')}
+                    </Typography>
+                  </StyledTableCell>
+                  {tableData.openingBalances?.map((balance, idx) => {
+                    return (
+                      <StyledTableCell key={`${idx}-${balance}`} align="right">
+                        {numberFormat(balance, locale)}
+                      </StyledTableCell>
+                    );
+                  })}
+                </TableRow>
+                {/* Income header */}
+                <TableRow sx={{ background: theme.palette.mpdxGreen.main }}>
+                  <StyledTableCell>
+                    <Typography variant="body1">{t('Income')}</Typography>
+                  </StyledTableCell>
+                  {tableData.periods?.map((_, idx) => {
+                    return (
+                      <StyledTableCell
+                        key={`income-${idx}`}
+                        align="right"
```

Comment:
This shouldn't need styles since it's empty, right?

Reply:
Thi table row (Opening Balance) isn't empty, I assume you mean the Income table row. This table row needs styles and empty rows to go all away across the table

Reply:
Sorry if my comment showed up in a weird place and wasn't clear. Yes, I meant the income header. I'm asking why we have `align="right"` on this cell and the expenses cell since the cell doesn't have any content?